docs/replication/group-replication-system-variables.md

```diff
@@ -76,4 +78,45 @@ The "MAJORITY" value is in [tech preview](../glossary.md#tech-preview) mode. Bef
 
 The variable specifies the mode use for flow control.
 
-*Percona Server for MySQL* 8.0.30-22 adds the "MAJORITY" value to the [group_replication_flow_control_mode](https://dev.mysql.com/doc/refman/8.0/en/group-replication-options.html#sysvar_group_replication_flow_control_mode) variable. In "MAJORITY" mode, [flow control](group-replication-flow-control.md) is activated only if the majority, more than half the number of members, exceed the flow control threshold. The other values are not changed. 
\ No newline at end of file
+*Percona Server for MySQL* 8.0.30-22 adds the "MAJORITY" value to the [group_replication_flow_control_mode](https://dev.mysql.com/doc/refman/8.0/en/group-replication-options.html#sysvar_group_replication_flow_control_mode) variable. In "MAJORITY" mode, [flow control](group-replication-flow-control.md) is activated only if the majority, more than half the number of members, exceed the flow control threshold. The other values are not changed.
+
+## group_replication_xcom_ssl_accept_retries
+
+| Option | Description |
+|---|---|
+| Introduced | 8.0.34-26 |
+| Command-line |--group_replication_xcom_ssl_accept_retries |
+| Dynamic | Yes |
+| Scope | Global |
+| Data type | integer |
+| Default value | 10 |
+| Measured in | seconds |
```

Comment:
A small correction: This variable is the actually the number of times SSL_accept() is called before closing the connection. So I don't think "Measured in seconds" is appropriate here.
```suggestion
| Measured in | seconds |
```